src/app/courses/components/course-card/course-card.component.html

```diff
@@ -1 +1,22 @@
-<p>course-card works!</p>
+<div class="card-container">
```

Comment:
You can use `<mat-card>` here. It will add a box-shadow, that is currently not implemented.

Reply:
fixed thanks

Reply:
Sorry, I was not clear here.
There should be a good reason for adding a library to your application. If you add angular material, it would be better to use its components. Adding such a big library as angular material only for using icons is not reasonable.

Course card is a perfect case for `mat-card`, so I suggest using it. Same with a modal dialog (you will create it later in the course).

To summarize: it's not about box-shadow. It's about using the library.

Reply:
Sorry, I was not clear here.

There should be a good reason to add a library to your application. If you add angular material, it would be better to use its components. Adding such a big library only for icons is not reasonable.

Course card is the perfect use case for `mat-card`, so I suggest using it. Same with modal-dialog component (you will create it later in the course).

To summarize: it's not about box shadow. It's about using a library.

Reply:
fixed